modules/ocf_syslog/files/logrotate

```diff
@@ -1,9 +1,10 @@
-/var/log/remote/* {
-	daily
-	missingok
-	rotate 14
-	compress
+/var/log/remote/*.log {
+    daily
+    missingok
+    rotate 14
+    compress
     delaycompress
+    create 0640 root ocfstaff
```

Comment:
Ah I see, I wondered why you left out `notifempty` too, but that makes a lot more sense now.

Sounds fine to me, looks like I can change some rsyslog stuff to get the same behavior without the issue with typos.